tests/testthat/test-utilities-static.R

```diff
@@ -6,6 +6,7 @@ test_that("callStatic returns a NetObject when the static .NET method returns a
   testObj <- callTestCase("CreateTestObjectGenericInstance")
   expect_true(inherits(testObj, "NetObject"))
   # 2DO: Test for IS!
+  # https://github.com/Open-Systems-Pharmacology/rSharp/issues/67
```

Comment:
Make a link in the issue to this line @PavelBal so the future dev who works on it can easily find this from the issue

Reply:
Done